confidential-data-hub/hub/src/bin/confidential-data-hub/config.rs

```diff
@@ -120,11 +120,15 @@ impl CdhConfig {
 
 impl CdhConfig {
     pub fn set_configuration_envs(&self) {
-        // KBS configurations
-        env::set_var(
-            "AA_KBC_PARAMS",
-            format!("{}::{}", self.kbc.name, self.kbc.url),
-        );
+        if let Err(_) = attestation_agent::config::aa_kbc_params::get_value() {
```

Comment:
Hi @Xynnn007 !

`attestation_agent::config::aa_kbc_params::get_value()` is a very simple function. I wonder if it wouldn't make sense to simply duplicate it to CDH to avoid creating this dependency CDH -> AA as the build of AA can get overly complicated.

Reply:
Never mind. I just noticed cdh -> AA dependency is already established.

Reply:
indeed, but they communicate via API and a clear interface, so a direct dep like this is unfortunate. I think we want to establish what configuration options we want to provide and then build configuration accordingly. At the moment this is more or less spec' ed in the implementation (e.g. "if aa_kbc_params kernel param is unset and there is no CDH config file, set aa_kbc_params as environment parameter to configure the CDH KMS - but without KBS cert").

So, there are a few assumptions, and it would be a good idea to simplify, even if that means duplication, e.g. the introduction of a `cdh_kbs_params` kernel param if we want to support passport on platforms without dynamic initdata.